howto/consume-topic/README.md

```diff
@@ -63,6 +63,24 @@ app.post('/orders', (req, res) => {
 app.listen(port, () => console.log(`consumer app listening on port ${port}!`))
 </pre>
 
+**NOTE:** For node/express > 4.16 you will need to add the following:
```

Comment:
These instructions should not be part of the example. Also, they might become stale in a future release on express (5.0, for example). We should stick to one version and just document it. Please, just rewrite this note to call out the express version and that the code might not work with future express versions.